openwisp_firmware_upgrader/api/urls.py

```diff
@@ -57,6 +57,11 @@
                     views.upgrade_operation_detail,
                     name="api_upgradeoperation_detail",
                 ),
+                path(
+                    "upgrade-operation/<uuid:pk>/cancel/",
+                    views.cancel_upgrade_operation,
+                    name="api_cancel_upgradeoperation",
+                ),
```

Comment:
let's maintain consistency
```suggestion
                path(
                    "upgrade-operation/<uuid:pk>/cancel/",
                    views.upgrade_operation_cancel,
                    name="api_upgradeoperation_cancel",
                ),
```